src/components/ProductCard.tsx

```diff
@@ -16,82 +17,57 @@ export default function ProductCard({ content }: { content: Content }) {
   const [selectedColour, setSelectedColour] = useState("black");
   const [cardImage, setCardImage] = useState(content.image[0]);
   const [isCardFlipped, setIsCardFlipped] = useState(true);
+  const [opacity, setOpacity] = useState(1);
 
-  useEffect(() => {
-    setCardImage(content.image[0]);
-  }, [content]);
+  const changeImage = (newImage: string) => {
+    setOpacity(0);
 
-  const toggleIsFlippedCard = () => {
+    setTimeout(() => {
+      setCardImage(newImage);
+      setOpacity(1);
+    }, 350);
+  };
+
+  const changeIsCardFlipped = () => {
```

Comment:
I think toggleIsFlippedCard is a better variable name than the new change

Reply:
addressed.